optimization/physical_constants.py

```diff
@@ -3,6 +3,102 @@
 """
 import math
 
+class FlowProperties:
+    """Class to store flow properties and calculate secondary properties from
+    fundamental props.
+    """
+    # coefficients for linear fit f(T) = A*T + b
+    coeffs = {'k'   : (7.0182e-5, 0.0135),
+              'mu'  : (2.6652e-8, 1.32523e-5),
+              'rho' : (-0.080314, 167.308),
+              'Cp'  : (0.255, 977.66),
+              # temperature limit for curve fits
+              'T_limits' : (900, 1200)
+             }
+    
+    W_e = 40000 # target electrical power [W]
+
+    ###############################################################################
+    #                                                                             #
+    #                     Given Parameters From Power Cycle                       #
+    #                                                                             #
+    ###############################################################################
+    m_dot = 0.75  # coolant flow [kg/s]
+    Q_therm = 131000  # core thermal power [W]
+    eta = W_e / Q_therm
+    T_in = 962.9  # core inlet temp [K] (from power cycle model)
+    T_out = 1100  # core outlet temp [K] (from power cycle model)
+    T = T_in + (T_out - T_in) / 2  # bulk coolant temp. [K]
+    rho = 86.96  # coolant density [kg/m^3]
+    k_cool = 0.086  # coolant conductivity [W/m-k]
+    Cp_cool = 1242  # coolant specific heat [J/kg-k]
+    mu = 0.00004079  # coolant viscosity [kg/m-s]
+    Pr = Cp_cool * mu / k_cool  # coolant Prandtl numbe[-]
+    P_in = 1.79064e7  # inlet pressure [Pa]
+    P_out = 1.74229e7  # outlet pressure [Pa]
+    dp_allowed = P_in - P_out  # pressure drop limit [Pa]
+    default = True
```

Comment:
What is it for ? checking if the values are different from the default values ?

what if I changed them to the default values ?

Reply:
it is actually never used.

Reply:
Does PEP8 say anything about this kind of initialization? It is a strange pattern to me, but I might just be outdated.  I would normally expect initialization to some defaults to happen in the constructor (`__init__`)...

Reply:
Especially when some of these are arguments of that constructor.